contents/docs/error-tracking/managing-issues.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Issue management
+---
+
+Our error tracking product gives you multiple ways to help you manage issues and work towards resolution. 
+
+## Assigning issues
+
+Issues can be assigned to a teammate from the list or issue page. To do this, select the **assignee** option and search for the teammate you want to assign it to, and select them. You can click this dropdown again to remove the assignee.
+
+What if you want to assign issues to a group rather than an individual teammate? You can create a role [your project settings](https://us.posthog.com/settings/organization-roles) under **Error tracking**.
+
+<ProductScreenshot
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/roles_light_6c7ea17be9.png"
+  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/roles_dark_f721b94577.png"
+  alt="Error tracking role assignees"
+  classes="rounded"
+/>
+
+### Auto assignment rules
+
+You can also automatically assign new and reopened issues using [auto assignment rules](https://us.posthog.com/error_tracking/configuration?tab=error-tracking-auto-assignment#selectedSetting=error-tracking-auto-assignment).
+Assignment conditions are evaluated against the properties of the exception event that created the issue.
+Because assignment rules are evaluated during ingestion the stack trace (if present) will be unminified, which allows for filtering on on exception properties such as function name and source file.
```

Comment:
```suggestion
You can automatically assign new and reopened issues using [auto assignment rules](https://us.posthog.com/error_tracking/configuration?tab=error-tracking-auto-assignment#selectedSetting=error-tracking-auto-assignment).

Assignment conditions are evaluated against the properties of the exception event that created the issue. Because assignment rules are evaluated during ingestion, the stack trace (if present) will be unminified which allows for filtering on on exception properties such as function name and source file.
```